backend/Tasque.Core.WebAPI/Controllers/OrganizationController.cs

```diff
@@ -17,7 +17,7 @@ public class OrganizationController : EntityController<Organization, Organizatio
         [HttpGet]
         public virtual IActionResult GetUserOrganizationsById(int userId)
         {
-            var organizations = _service.GetUserOrganizations(userId);
+            var organizations = _service.GetUserOrganizations(userId).Result;
```

Comment:
Should make method async and use await here

Reply:
Done.